tests/content_api_test.py

```diff
@@ -574,3 +575,89 @@ def test_publish_item_with_attachments(self):
         with self.capi.test_client() as c:
             response = c.get(attachments[0]['href'], headers=self._auth_headers(subscriber))
             self.assertEqual(200, response.status_code, attachments[0]['href'])
+
+    def test_items_default_sorting(self):
+        subscriber = {'_id': 'sub1'}
+        headers = self._auth_headers(subscriber)
+
+        self.content_api.publish(
+            {'_id': 'aaaa', 'guid': 'aaa', 'urgency': '1', 'type': 'text'},
+            [subscriber]
+        )
+        time.sleep(1)
```

Comment:
why doing the sleep? I'm a bit afraid this will make testing too long.

Reply:
@jerome-poisson I agree that it is a little bit weird :) The trick is that we want to test a default sorting and default sort field is `versioncreated`, so if I `self.content_api.publish` without delay `versioncreated` will be almost the same (it filled automatically in publish service). The lowest dimension is a `second` in datetime for versioncreated. That's why I use delay with 1 second. Does it make any sense?

Reply:
OK it makes sense. Can you add a comment to explain that?

Reply:
sure